portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Khadija's Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+    <p> Hello! My name is Khadija and I am a rising junior at Pomona College majoring in 
+          Computer Science and Africana Studies. I am originally from NY but now I go to school in California(the weather is usually amazing, but it does get cold at times).
+          I love music, especially Soul and RnB and want to learn the electric guitar. I am also a reader, I love reading all types of books and have been investing alot of time in that. </p>
+     <p> Projects </p>
+      <p> I do not have many projects but these are a few things I have worked on</p>
+     <ul> 
+         <li> Hackathon App</li>
+         <li> Social Issues in Computer Sciene Classrooms: Designed a final project for computer science intro course, pertaining to social issue. </li>
+         <li> Building Autotester for school website</li>
+     </ul>
+      <p> A favorite Photo of mine. The colors, design and such really intrigued me. </p> 
```

Comment:
Nice to see the portfolio content.  Just pay attention to the indentation of the code since it improves readability